ilios/frontend
# Publish all sessions

WIP

I’m starting a PR to get feedback on the overall functionality and display.  @saschaben can you take a look?

Steps to test -
1) Use http://56247d26d6865d4b8b000005.zoo-keeper-levels-51188.netlify.com/ as proxy wont work until #888 is merged
2) Pick a course with sessions and click the ugly green button in the session header
3) test, test, test

Questions I have:
Do we need this much information on this screen?
Would the info make more sense if we removed everything that was done and only displayed what is missing?
Do the Publish As Is / Mark As Scheduled checkboxes make sense?
Do the checkboxes in the table header to publish all as is and mark all as scheduled make sense?
I'm planning to collapse the unpublishable and publishable sections.  I think if they are collapsed they belong on top of the section where you can make publish / mark decisions.  Is that correct?
How can we make it clear that when you click save you are not only saving those sessions that you acted upon but also publishing everything in the publishable section?
Where should the save button go?  I can put more than one on the screen if it needs to be in a couple of places.
Other thoughts on function, other than the fact that it is pretty ugly right now?
Fixes #797
